AutoLab.sln

```diff
@@ -13,7 +13,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Homework5_Lib", "Homework5_
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Homework5_Main", "Homework5_Main\Homework5_Main.csproj", "{D6A57EB0-F240-473E-87DF-9684DDDC3B8F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Homework7", "Homework7\Homework7.csproj", "{3CF9BB53-791C-4B94-B1D6-31A8D243865A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Homework7", "Homework7\Homework7.csproj", "{3CF9BB53-791C-4B94-B1D6-31A8D243865A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Homework8_1", "Homework8\Homework8_1.csproj", "{9738E72A-22DE-47CE-AF63-CF48A3F3F2AD}"
```

Comment:
Still problem with loading this project
![image](https://user-images.githubusercontent.com/23722661/203250794-79a2140c-404d-4f41-bf05-ec84c1b55410.png)
You need to change folder name to correct
![image](https://user-images.githubusercontent.com/23722661/203250945-685b2940-3f1d-47a4-98f2-c43aa8a4675f.png)

Reply:
hope that fixed